### DISCUSSION TITLE
Add @Cito as developer

### DISCUSSION THREAD

**Discussion by superDuperCyberTechno** [superDuperCyberTechno/obsidian-code-copy#7] [Ideas]
@Cito, I'm not proficient enough with the JS infrastructure to be spending any meaningful time troubleshooting my build environment right now. Would you mind if I added you as a permanent contributor?

I merged your PR, all we need is to build and release 1.0.4.

You're free to add stuff, just please swing them by me first.

**Comment by Cito** [comment]
Thanks. I don't have much experience with obsidian plugin development yet and am not sure how much time I will find for this in the future, but feel free to add me as developer.
